common/moduleconfig/testdata/projects/sample/sample.go

```diff
@@ -0,0 +1,21 @@
+package sample
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/TBD54566975/ftl/go-runtime/ftl" // Import the FTL SDK.
+)
```

Comment:
Was this module added to test this new version stuff? If so, maybe we can give it a more descriptive name? I'm ok with sample too given that we have others named like `one`, `two` etc. But if there's an obvious name to use here that would be preferred